pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -1,15 +1,21 @@
-{ linkFarmFromDrvs, fetchurl }:
+{ linkFarmFromDrvs, substituteAll, fetchurl, runCommand, zip, dotnet-sdk }:
 { name, nugetDeps ? import sourceFile, sourceFile ? null }:
 linkFarmFromDrvs "${name}-nuget-deps" (nugetDeps {
   fetchNuGet = { pname, version, sha256 ? "", hash ? ""
     , url ? "https://www.nuget.org/api/v2/package/${pname}/${version}" }:
-    fetchurl {
-      name = "${pname}.${version}.nupkg";
+    let from-source = fetchurl {
```

Comment:
I think this might be more intuitive as something like:

```
stdenvNoCC.mkDerivation {
  name = [...];
  src = fetchurl [...];
  nativeBuildInputs = [ zip ];
  installPhase = "cp $src $out";
  fixupPhase = "zip -d $out .signature.p7s";
}

Reply:
Fair enough - when I wrote it this way it was in anticipation of this being more involved than it ended up being.